.github/workflows/main.yml

```diff
@@ -16,8 +16,8 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: "14"
-          cache: "npm"
+          node-version: '14'
```

Comment:
I was surprised to see so many code changes. But I suspect you have set up Prettier in your IDE with different rules, right? I would stick to the existing conventions. Would be interested what caused this reformating

Reply:
Fyi this will also fix the CI pipeline that's currently failing. If this whole Git workflow with CI pipeline is new to you I can also recommend my free [GitHub Minesweeper course](https://profy.dev/project/github-minesweeper). Should only take an hour or two

<img width="859" alt="grafik" src="https://user-images.githubusercontent.com/9722792/195248924-c6b83d18-a793-4fe2-ac35-9f3b4c04743b.png">

Reply:
I see now that you tried to fix the problem in the other PRs but it didn't seem to work out. Can you tell me what editor/IDE you're using? Likely other devs will run into the same problem as well so it would be helpful to understand what's going on